lib/src/model.dart

```diff
@@ -1605,6 +1611,27 @@ abstract class ModelElement implements Comparable, Nameable, Documentable {
 
     return '<a ${classContent}href="${href}">$name</a>';
   }
+
+  String _injectExamples(String rawdocs) {
+    if (rawdocs.contains('@example')) {
+      RegExp exp = new RegExp(r"{@example .+}");
+      Iterable<Match> matches = exp.allMatches(rawdocs);
+      var dirPath = this.package.packageMeta.dir.path;
+      for (var match in matches) {
+        var strings = match.group(0).split(' ');
+        var file = new File(p.join(dirPath, 'examples', strings[1]));
+        if (file.existsSync()) {
+          // TODO(keertip):inject example
```

Comment:
Yes. Currently there are no examples in Angular2 dart, getting this in will give them a list of the missing examples.